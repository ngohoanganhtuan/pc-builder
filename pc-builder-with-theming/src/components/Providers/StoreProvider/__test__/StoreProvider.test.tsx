import renderer from 'react-test-renderer';

// Components
import { StoreProvider } from '@components';

describe('Test [StoreProvider] component', () => {
  test('Component [StoreProvider] should render correctly', () => {
    const component = renderer
      .create(
        <StoreProvider>
          <h1>React Children</h1>
        </StoreProvider>
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
