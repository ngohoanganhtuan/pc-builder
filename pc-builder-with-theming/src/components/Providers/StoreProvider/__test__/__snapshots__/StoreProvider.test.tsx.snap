// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test [StoreProvider] component Component [StoreProvider] should render correctly 1`] = `
<h1>
  React Children
</h1>
`;
